File: openapi-test-kotlin-server-quarkus-reactive-client-mp-reactive/testapi/model/Bird.kt
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import jakarta.validation.constraints.NotNull
import java.time.OffsetDateTime
import java.util.Optional

data class Bird(
  @JsonProperty("id") @NotNull val id: String,
  @JsonProperty("createdAt") @NotNull val createdAt: OffsetDateTime,
  @JsonProperty("name") @NotNull val name: String,
  @JsonProperty("updatedAt") val updatedAt: Optional<OffsetDateTime>,
  @JsonProperty("wingSpan") val wingSpan: Optional<Double>,
  @JsonProperty("canFly") @NotNull val canFly: Boolean
) : Animal {
  override fun animal_type(): String = "bird"
}

File: openapi-test-scala-server-spring-client-jdk/testapi/model/Bird.scala
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.OffsetDateTime
import java.util.Optional

case class Bird(
  @JsonProperty("id") id: PetId,
  @JsonProperty("createdAt") createdAt: OffsetDateTime,
  @JsonProperty("name") name: String,
  @JsonProperty("updatedAt") updatedAt: Optional[OffsetDateTime],
  @JsonProperty("wingSpan") wingSpan: Optional[java.lang.Double],
  @JsonProperty("canFly") canFly: java.lang.Boolean
) extends Animal {
  @JsonProperty("animal_type")
  override lazy val animal_type: String = "bird"
}